common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -765,18 +836,18 @@ private void processCloudRedirectAndPrtHeader(@NonNull final WebView view, @NonN
         final SpanContext spanContext = getActivity() instanceof AuthorizationActivity ? ((AuthorizationActivity) getActivity()).getSpanContext() : null;
         final Span span = spanContext != null ?
                 OTelUtility.createSpanFromParent(SpanName.ProcessCrossCloudRedirect.name(), spanContext) : OTelUtility.createSpan(SpanName.ProcessCrossCloudRedirect.name());
-        final CrossCloudChallengeHandler crossCloudChallengeHandler = new CrossCloudChallengeHandler(view, mRequestHeaders, span);
-        processCloudRedirectAndPrtHeaderInternal(url, crossCloudChallengeHandler, view, methodTag, span);
+        final ReAttachPrtHeaderHandler reAttachPrtHeaderHandler = new ReAttachPrtHeaderHandler(view, mRequestHeaders, span);
```

Comment:
For line 833: the method probably needs to be renamed

Reply:
I think method processCloudRedirectAndPrtHeader can be renamed as processCrossCloudRedirect because we are adding spans that specifically capture that it is a cross cloud redirect and then attach prt header. This may not be needed in the future once we start attaching PRT headers for every entra redirect URL. But for now, I am keeping the method name to reflect that it is processing a cross cloud redirect.